commands/lint.js

```diff
@@ -52,6 +67,52 @@ function api({strict = false, local = false, fix = false, eslintArgs = []} = {})
 	});
 }
 
+function tslintBin(context) {
+	try {
+		resolver.sync('tslint', {basedir: context});
+		return path.join(context, 'node_modules', '.bin', 'tslint');
+	} catch (e) {
+		return 'tslint';
+	}
+}
+
+function shouldTSLint(context) {
+	if (fs.existsSync(path.join(context, 'tslint.json'))) {
+		if (glob.sync('**/*.+(ts|tsx)', globOpts).length > 0) {
+			try {
+				return !spawn.sync(tslintBin(context), ['-v'], {stdio: 'ignore'}).error;
+			} catch (e) {
+				// ignore
+			}
+		}
+	}
+	return false;
+}
+
+function tslint({fix = false} = {}, context) {
+	const args = ['-p', context];
+	if (fix) args.push('--fix');
```

Comment:
Should we push the eslintargs (and rename that prop to be something more generic)?

Reply:
I'd say no, since eslint is still being run on any js files and has a different usage-case. If you're looking for something to pass TSLint options, could add an unused programatic option to `api()`, but I'm of the belief that `enact lint` is a shortcut for common-usage, not an outright replacement for TSLint (since unlike ESLint, we have no embedded Enact tslint config and it's user-configurable/opt-in from app-side rather than CLI-side)

Reply:
Makes sense!